com.remnantsoftheprecursors.ROTP.yml

```diff
@@ -0,0 +1,55 @@
+app-id: com.remnantsoftheprecursors.ROTP
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: Remnants.sh
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: 
+    - /usr/lib/sdk/openjdk17/install.sh
+  - name: rotp
+    buildsystem: simple
+    build-options:
+      env:
+        PATH: ${PATH}:/app/bin:/usr/bin:/usr/lib/sdk/openjdk17/bin
+        MAVEN_OPTS: -Dmaven.repo.local=.m2/repository
+        JAVA_HOME: /usr/lib/sdk/openjdk17/jvm/openjdk-17
+    build-commands:
+    - patch -u src/rotp/Rotp.java -i startupDir.patch
+    - mvn compile
+    sources:
+    - type: git
+      url: https://github.com/rayfowler/rotp-public.git
+      commit: 55bf0b837a27ab61796e1320932186141494c777
+    - type: file
+      path: pom.xml
+    - type: file
+      path: startupDir.patch
+    - maven-dependencies.yaml
+  - name: rotp-misc
+    buildsystem: simple
+    build-commands:
+    - install -D Remnants.sh /app/bin/Remnants.sh
+    - install -D com.remnantsoftheprecursors.ROTP.png /app/share/icons/hicolor/128x128/apps/com.remnantsoftheprecursors.ROTP.png
+    - install -D com.remnantsoftheprecursors.ROTP.desktop /app/share/applications/com.remnantsoftheprecursors.ROTP.desktop
+    - install -D com.remnantsoftheprecursors.ROTP.metainfo.xml /app/share/metainfo/com.remnantsoftheprecursors.ROTP.metainfo.xml
+    sources:
+    - type: file
+      path: Remnants.sh
+    - type: file
+      path: com.remnantsoftheprecursors.ROTP.png
+    - type: file
+      path: com.remnantsoftheprecursors.ROTP.desktop
+    - type: file
+      path: com.remnantsoftheprecursors.ROTP.metainfo.xml
+finish-args:
```

Comment:
finish-args are usually toward the top before the `modules`

Reply:
Sure! I was following the openjdk example.. my bad